data/campaigns/Under_the_Burning_Suns/units/quenoth/Tauroch_Stalwart.cfg

```diff
@@ -15,38 +15,42 @@
     {TRAIT_RESILIENT}
     {TRAIT_DEXTROUS}
     {TRAIT_HEALTHY}
-    hitpoints=56
+    hitpoints=48
```

Comment:
Again big gigantic nerf, are you sure about this?

Reply:
You are the one that asked me to do this because they felt too strong when you played them.